cmd/dptp-controller-manager/main.go

```diff
@@ -474,6 +472,9 @@ func main() {
 		if opts.registrySyncerOptions.imagePullSecretPath == "" {
 			logrus.Fatal("The registrysyncer requires the --registrySyncerOptions.imagePullSecretPath flag to be set ")
 		}
+		if _, hasApiCI := kubeconfigs[apiCIContextName]; !hasApiCI {
```

Comment:
why do we use `apiCIContextName` and not `opts.registryClusterName` here?

Reply:
Because `opts.registryClusterName` could be a cluster other than api.ci in the future and we do registry sync for app.ci and api.ci for the moment which still holds after migrating the registry.